pkcs11/_pkcs11.pyx

```diff
@@ -235,6 +235,13 @@ class Token(types.Token):
 
         if user_pin is not None and so_pin is not None:
             raise ArgumentsBad("Set either `user_pin` or `so_pin`")
+        elif user_pin is not None and use_pap:
```

Comment:
What is this variable?

Reply:
This is a mistake--it should be the p_auth variable.